setup.cfg

```diff
@@ -0,0 +1,10 @@
+# [tool:pytest]
```

Comment:
Do we need this file?

Reply:
fixed in 6cec3ca1417d268a544349d1f4a3c9ebfb386912